core/vm/exit_code/exit_code.hpp

```diff
@@ -0,0 +1,182 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_CORE_VM_EXITCODE_EXITCODE_HPP
+#define CPP_FILECOIN_CORE_VM_EXITCODE_EXITCODE_HPP
+
+#include "boost/variant.hpp"
+
+namespace fc::vm::exit_code {
+
+  /**
+   * @brief Successful exit code
+   */
+  struct Success {
+    /**
+     * to be EqualityComparable, always return true
+     * @param other - other Success object
+     * @return true
+     */
+    bool operator==(const Success &other) const;
+  };
+
+  /**
+   * @brief System error codes
+   */
+  enum class SystemError {
+    // Represents a failure to find an actor.
```

Comment:
Since those are docstrings, let's use /// or /** */ for multiline strings here and below.